src/iterative_sorting/iterative_sorting.py

```diff
@@ -2,25 +2,28 @@
 def selection_sort( arr ):
     # loop through n-1 elements
     for i in range(0, len(arr) - 1):
-        cur_index = i
-        smallest_index = cur_index
         # TO-DO: find next smallest element
         # (hint, can do in 3 loc) 
-             
-
-
-
-        # TO-DO: swap
-
-
-
-
+        min_val = min(arr[i:])
```

Comment:
Oh snap, I haven't seen this solution before, this is pretty awesome! I wonder whether it might be less efficient that stepping through the array once, because the functions to find the minimum and then find the index might run through the array twice, but that's still O(n) :man_shrugging: